<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GifAnimatorDialog</class>
 <widget class="QDialog" name="GifAnimatorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gif Animator</string>
  </property>
  <widget class="QPushButton" name="pushButtonAnimate">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Gif Animation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOpenFolder">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Output Folder</string>
   </property>
   <property name="icon">
    <iconset resource="ExMplayer.qrc">
     <normaloff>:/images/fileopen.png</normaloff>:/images/fileopen.png</iconset>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>297</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButtonStart">
      <property name="text">
       <string>Mark Start</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditStart"/>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButtonReset">
      <property name="text">
       <string>Reset</string>
      </property>
      <property name="icon">
       <iconset resource="ExMplayer.qrc">
        <normaloff>:/images/reset.png</normaloff>:/images/reset.png</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButtonStop">
      <property name="text">
       <string>Mark Stop</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditStop"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>101</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ExMplayer.qrc">:/images/ExManim.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>411</width>
     <height>20</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="labelLoader">
      <property name="maximumSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="ExMplayer.qrc">:/images/loaderanim.gif</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelStatus">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="ExMplayer.qrc"/>
 </resources>
 <connections/>
</ui>
